mandatory/3-function-output.js

```diff
@@ -1,14 +1,22 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+/* Math.random() returns a random number between 0 (inclusive),  and 1 (exclusive). this  code generate random number between 0 and 1 then multiply by 10. */
+
 function getRandomNumber() {
   return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+
+/* The concat() method is used to merge two or more arrays. This method does not change the existing arrays, but instead returns a new array. */
+
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
+  return firstWord.concat(" ", secondWord, " ", thirdWord);
```

Comment:
This looks good.
Can you thing of any other ways to do this?
Here are some ideas: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
thanks for your feed back @moneyinthesky . i will look at this